react/components/atoms/text-input/text-input.js

```diff
@@ -0,0 +1,90 @@
+import React, { PureComponent } from "react";
+import { StyleSheet, Text, TextInput, View } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { IdentifiableMixin, baseStyles } from "../../../util";
+
+export class TextInputRipe extends mix(PureComponent).with(IdentifiableMixin) {

```

Comment:
I'm not sure if it should be called TextInput, maybe FormInput, to make it similar to components-vue component